recipe/meta.yaml

```diff
@@ -17,20 +18,24 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - pytest-runner
+    - wheel
     - setuptools >=30.0.0
   run:
     - python >=3.6
     - mccabe >=0.6.0,<0.7.0
-    - pycodestyle >=2.7.0,<2.8.0
-    - pyflakes >=2.3.0,<2.4.0
+    - pycodestyle >=2.8.0,<2.9.0
+    - pyflakes >=2.4.0,<2.5.0
     - importlib-metadata  # [py<38]
```

Comment:
We should remove pinning a dependency in noarch packages:
` - importlib-metadata`

Because of excluding this dependency in the final artifact and causing issues when users will install flake8 without `importlib-metadata` with `py<38` https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=meta.yaml#architecture-independent-packages

Reply:
Sorry, I mean selectors and not pinning. Sometimes it's pretty confusing